drbd-formula.changes

```diff
@@ -1,3 +1,16 @@
+-------------------------------------------------------------------
+Tue Jul 27 11:41:34 UTC 2021 - Eike Waldt <waldt@b1-systems.de>
+
+- Version bump 0.5.0
+  * Make virtual_ip for NFS service mandatory
+
+-------------------------------------------------------------------
+Thu Jul 22 09:54:20 UTC 2021 - Eike Waldt <waldt@b1-systems.de>
```

Comment:
Do we need this intermediate entry?
I guess that it was created during this implementation, but I would just remove it and squash all the entries in the 0.5.0 version